client/api.go

```diff
@@ -37,6 +72,36 @@ func (x *coreServer) netMapSnapshot(ctx context.Context, req v2netmap.SnapshotRe
 	return resp, nil
 }
 
+// executes NetmapService.NetworkInfo RPC declared in NeoFS API protocol
+// using underlying client.Client.
+func (x *coreServer) getNetworkInfo(ctx context.Context, req netmap.NetworkInfoRequest) (*netmap.NetworkInfoResponse, error) {
+	resp, err := rpcapi.NetworkInfo((*client.Client)(x), &req, client.WithContext(ctx))
+	if err != nil {
+		return nil, rpcErr(err)
```

Comment:
Maybe it's easier to teach `rpcErr` handling `nil`? So that it'd just be `return resp, rpcErr(err)`. Although the code is already there and it will work of course.

Reply:
was reasonable for this commit, but i'd need to teach it back

Reply:
Yeah, it's changed significantly afterwards, so doesn't matter in the end.